### PR TITLE
media-sound/pnmixer: keyworded 0.7.1 for ppc, bug #657232

### PR DESCRIPTION
**Pull request by ernsteiswuerfel**
keyworded 0.7.1 for ppc
bumped EAPI from 5 to 6 in 0.6_pre20111213 without addidtional changes

Closes: https://bugs.gentoo.org/657232
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: media-sound/pnmixer/pnmixer-0.6_pre20111213.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
```
Comment (mgorny) [inline media-sound/pnmixer/pnmixer-0.6_pre20111213.ebuild]: Please don't change EAPI of stable ebuilds. Only bump the newest version, and if it's stable (or EAPI change is likely to break stuff), then you need to create a revision bump.

File: media-sound/pnmixer/pnmixer-0.7.1.ebuild [hunk 2]
```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/nicklan/pnmixer/archive/v${PV/_rc/-rc}.tar.gz -> ${P
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~x86 ~ppc"
```
Comment (mgorny) [inline media-sound/pnmixer/pnmixer-0.7.1.ebuild]: Please keep keywords sorted in `ekeyword`-ish way. Preferably just use `ekeyword` tool to update keywords.

Comment (ernsteiswuerfel) [reply media-sound/pnmixer/pnmixer-0.7.1.ebuild]: Thanks for your feedback! Hope I got that right now. Had some help on IRC. ;) I will of course take your advice in accout for future PRs.

**Comment by mgorny** [comment]
Thank you. I'll squash the commits for you and merge it.

**Pull request closed without merging**